src/components/Tabs/components/TabList/TabList.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import styled, { css } from 'react-emotion';
+import { styleHelpers } from '../../../../styles';
+
+import Tab from '../Tab';
+
+const MOBILE_AUTOSTRETCH_ITEMS_LIMIT = 4;
```

Comment:
Out of curiosity, how did you pick this number?

Reply:
I'm curious also to see how that affects the UI for the responsive version! 👀

Reply:
When there are three or less tabs, they get stretched 100% on mobile and are aligned on both sides (left/right). Looks better. Also added the option to use a prop for that to force it on desktops as well (if there is need).

Reply:
I got that part, what I'm wondering is why it's exactly 4? This might work great for single, short English words as tab label, but likely doesn't work for languages with long words, e.g. Finnish or German. Or does it? Does the tab label wrap?

Reply:
The max is actually 3 (33% per piece). The number is 4, because the comparison is `items.length < 4`. I think I've picked the wrong representation/naming here :)

Will change.